src/Posting/QR.cooldown.coffee

```diff
@@ -105,6 +105,10 @@ QR.cooldown =
             seconds = Math.max seconds, cooldown.delay - elapsed
           continue
 
+        # Increase the max delay if additional cooldown is enabled
+        if QR.cooldown.addCooldown
+          QR.cooldown.maxDelay = QR.cooldown.maxDelay + parseInt(Conf['additionalCooldown'])
```

Comment:
Here you want to change `maxDelay` instead of repeatedly increading `QR.cooldown.maxDelay`.